CHANGELOG.rst

```diff
@@ -12,6 +12,9 @@ New Features
 
   - deprecated ``validate``
 
+Improvements
+~~~~~~~~~~~~
+- Added possibility to query Edge IDs and Node IDs based on edge/node population type using query key ``population_type``
```

Comment:
It would be worth to mention that the ``population_type`` corresponds to ``type`` in the [sonata specs](https://sonata-extension.readthedocs.io/en/latest/sonata_config.html).

I also feel that the group concept could be documented more, because there is a "Query system" section in [From bluepy to bluepysnap](https://bbpgitlab.epfl.ch/nse/bluepy/-/blob/main/examples/ipython_notebooks/From%20bluepy%20to%20bluepysnap.ipynb) but it's not public.

For example we could add other examples in [Group Concept](https://github.com/BlueBrain/snap/blob/c9162f7373c79975f5e5232bd2896e9e2b50013e/bluepysnap/nodes/node_population.py#L20) using ``population_type``,``population`` and ``node_id``.

Reply:
I agree, I think it would also be beneficial to demo the different query options in the notebooks (I'll create a separate PR for that). 

Meanwhile, I'll update the docstring(s) and the change log.

Reply:
I added examples of `population_type` in the notebooks, that were added in #261 .